.codespell-ignorelines

```diff
@@ -0,0 +1 @@
+* Atelier de la donnée ADOC Lorraine (projet financé avec le soutien du Fonds national pour la Science ouverte)
```

Comment:
Can't this file be replaced by command line argument `--ignore-regex` instead? With a value of `/\bprojet financé\b/` or something?

I want to avoid the dotfile pollution of root dir.

Reply:
ok, done, pending push